examples/jsm/nodes/accessors/SkinningNode.js

```diff
@@ -17,7 +17,9 @@ import {
 
 import { NodeUpdateType } from '../core/constants.js';
 
-const Skinning = new ShaderNode( ( inputs, builder ) => {
+const Skinning = ( inputs, builder ) => {
+	
+	Object.keys( inputs ).forEach( key => inputs[ key ] = nodeObject( inputs[ key ] ) );
```

Comment:
I liked the previous approach here...

Reply:
I think that a possible solution would be to split ShaderNode into two parts - one containing ShaderNodeObject, ShaderNodeProxy, ShaderNode, nodeObject, and type conversion functions, and other one containing node proxies...

Reply:
Is it better now?